net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.inc

```diff
@@ -77,17 +77,17 @@ function quagga_ospfd_install_conf() {
 	// generate ospfd.conf based on the assistant
 	if (is_array($config['installedpackages']['quaggaospfd']['config'])) {
 		$ospfd_conf = &$config['installedpackages']['quaggaospfd']['config'][0];
-	} elseif (isset($config['installedpackages']['quaggaospfdraw']['config'][0]['ospfd'])  ||
-	    isset($config['installedpackages']['quaggaospfdraw']['config'][0]['ospf6d']) ||
-	    isset($config['installedpackages']['quaggaospfdraw']['config'][0]['bgpd'])) {
+	} elseif (isset($config['installedpackages']['quaggaospfdraw']['config'][0]['ospfd'])
+        || isset($config['installedpackages']['quaggaospfdraw']['config'][0]['ospf6d'])
+        || isset($config['installedpackages']['quaggaospfdraw']['config'][0]['bgpd'])) {
```

Comment:
The indentation here is not right. It should be a tab then four space. It appears to be all spaces.